### PR TITLE
add possibility to set east west domain

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by hjacobs** [comment]
Thanks.

**Comment by aryszka** [comment]
:+1:

File: dataclients/kubernetes/kube.go [hunk 1]
```diff
@@ -168,6 +168,9 @@ type Options struct {
 	// PathMode controls the default interpretation of ingress paths in cases when the ingress doesn't
 	// specify it with an annotation.
 	PathMode PathMode
+
+	// KubernetesEastWestDomain set the DNS domain to be used for east west traffic, defaults to "skipper.cluster.local"
```
[the remark this answers is not in the document]
Comment (szuecs) [reply dataclients/kubernetes/kube.go]: fixed

File: cmd/skipper/main.go [hunk 2]
```diff
@@ -146,6 +146,7 @@ const (
 	kubernetesPathModeUsage          = "controls the default interpretation of Kubernetes ingress paths: kubernetes-ingress/path-regexp/path-prefix"
 	kubernetesNamespaceUsage         = "watch only this namespace for ingresses"
 	kubernetesEnableEastWestUsage    = "enables east-west communication, which automatically adds routes for Ingress objects with hostname <name>.<namespace>.skipper.cluster.local"
+	kubernetesEastWestDomainUsage    = "set the east-west domain to the given"
```
[the remark this answers is not in the document]
Comment (szuecs) [reply cmd/skipper/main.go]: fixed

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**